lib/readable_streambuffer.js

```diff
@@ -42,6 +42,9 @@ var ReadableStreamBuffer = module.exports = function(opts) {
     if (sendMore) {
       sendData.timeout = setTimeout(sendData, frequency);
     }
+    else {
+      sendData.timeout = null;
```

Comment:
I don't know, this patch solved my issue but I've haven't actively used this package in some time.